operations/mimir-mixin/config.libsonnet

```diff
@@ -658,6 +658,12 @@
     // Set to four times the scrape interval to account for edge cases: https://www.robustperception.io/what-range-should-i-use-with-rate/
     recording_rules_range_interval: '1m',
 
+    // Used to calculate range interval in alerts.
+    // In seconds without the `s` at the end, i.e. `30`.
+    // Needed to account for edge cases: https://www.robustperception.io/what-range-should-i-use-with-rate/
+    scrape_interval: 15,
```

Comment:
i'd even drop this. My intention with it was to have it more clear what it relates to,  buut we already have `recording_rules_range_interval` which doesn't use that.